ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -15,6 +101,7 @@ def getChangeLogText() {
 }
 
 def call(BUILD_BRANCH, CREDENTIALS_ID, CODECOV_CREDENTIALS_ID) {
+	timeout(120) {
```

Comment:
Must be completed in two minutes?

Reply:
2 hours, unit is minutes

Reply:
Can we set the time unit explicitly? Like `timeout (time: 2, unit: 'HOURS')`?

Reply:
done